ocaml/idl/datamodel.ml

```diff
@@ -6007,6 +6007,7 @@ let storage_operations =
           "vdi_mirror", "Mirroring a VDI";
           "vdi_enable_cbt", "Enabling changed block tracking for a VDI";
           "vdi_disable_cbt", "Disabling changed block tracking for a VDI";
+          "vdi_hidden_cbt_setter", "Set cbt_enabled status in DB for a VDI";
```

Comment:
Let's not add this operation to `storage_operations` and `vdi_operations`, because it is always allowed. This is an internal call used by SM, so I don't think we need to do any locking (which is provided by `vdi_operations`), because SM should know when it's OK to call this.

Reply:
Just to be clear are you saying this should be in neither storage_operations and vdi_operations? or just in vdi_operations?

Reply:
Yes, it shouldn't be in either of those in my opinion.